src/index.html

```diff
@@ -1,14 +1,233 @@
+  acts which have outraged the conscience of mankind, and the advent of a world
+  in which human beings shall enjoy freedom of speech and belief and freedom
+  from fear and want has been proclaimed as the highest aspiration of the common
+  people,
+      <br>
+  Whereas it is essential, if man is not to be compelled to have recourse, as a last
+  resort, to rebellion against tyranny and oppression, that human rights should be
+  protected by the rule of law,
+      <br>
+  Whereas it is essential to promote the development of friendly relations between
+  nations,
+      <br>
+  Whereas the peoples of the United Nations have in the Charter reaffirmed their
+  faith in fundamental human rights, in the dignity and worth of the human person
+  and in the equal rights of men and women and have determined to promote
+  social progress and better standards of life in larger freedom,
+      <br>
+  Whereas Member States have pledged themselves to achieve, in cooperation
+  with the United Nations, the promotion of universal respect for and observance of
+  human rights and fundamental freedoms,
+      <br>
+  Whereas a common understanding of these rights and freedoms is of the
+  greatest importance for the full realization of this pledge,
+      <br>
+  Now, therefore,
+      <br>
+  The General Assembly,
+      <br>
+  Proclaims this Universal Declaration of Human Rights as a common standard of
+  achievement for all peoples and all nations, to the end that every individual and
+  every organ of society, keeping this Declaration constantly in mind, shall strive by
+  teaching and education to promote respect for these rights and freedoms and by
+  progressive measures, national and international, to secure their universal and
+  effective recognition and observance, both among the peoples of Member States
+  themselves and among the peoples of territories under their jurisdiction.
+    </p>
+
+    <h2 class="ba-ArtTitle">Article I</h2>
+
+    <p class="ba-par">
+  All human beings are born free and equal in dignity and rights. They are
+  endowed with reason and conscience and should act towards one another in a
+  spirit of brotherhood.
+    </p>
+
+    <h2 class="ba-ArtTitle">Article 2</h2>
+
+    <p class="ba-par">
+  Everyone is entitled to all the rights and freedoms set forth in this Declaration,
+  without distinction of any kind, such as race, colour, sex, language, religion,
+  political or other opinion, national or social origin, property, birth or other status.
+  Furthermore, no distinction shall be made on the basis of the political,
+  jurisdictional or international status of the country or territory to which a person
+  belongs, whether it be independent, trust, non-self-governing or under any other
+  limitation of sovereignty.
+    </p>
+
+    <h2 class="ba-ArtTitle">Article 3</h2>
+
+    <p class="ba-par">
+  Everyone has the right to life, liberty and the security of person.
+    </p>
```

Comment:
correct indentation for text everywhere in the file

```suggestion
    <p class="ba-par">
       Everyone has the right to life, liberty and the security of person.
    </p>
```